tests/aws-cpp-sdk-dynamodb-integration-tests/TableOperationTest.cpp

```diff
@@ -100,6 +100,8 @@ class TableOperationTest : public ::testing::Test {
     std::mutex updateItemResultMutex;
     std::condition_variable updateItemResultSemaphore;
 
+    std::vector<Aws::String> m_tablesCreated;
```

Comment:
Tracks tables created, which gets cleaned up in the fixture destructor invocation